### ISSUE TITLE
rfswarm-manager fails to start when --nogui flag set.

### ISSUE THREAD

**Issue by MathiasWasAlreadyTaken** [damies13/rfswarm#81]
**Describe the bug**
When starting the manager (v0.6.[3,4]) with the --nogui flag, the manager is started twice. This results in the port is already started for the second instance, and in turn exits the manager.

**To Reproduce**
1. Start manager with --nogui flag: rfwarm-manager -s test.rfs --nogui --agents 2
2. See error

**Expected behavior**
Manager should wait for expected number of agents to connect

**Desktop (please complete the following information):**
 - OS: macOS BigSur 11.2.3
 - Python 9.3.1

**Comment by MathiasWasAlreadyTaken** [comment]
Changing the entry_points to:
    entry_points = {'console_scripts': ['rfswarm = rfswarm_manager.rfswarm', 'rfswarm-manager = rfswarm_manager.rfswarm']},
...made it work.
Not initiating the RFSwarmCore class I guess...I'm not an expert...

**Comment by damies13** [comment]
I'd like to thank Mathias for spending the time on discord to work through this issue with me and help me reproduce it and find the cause .

I reproduced it on my windows 10 VM
```
C:\Users\Dave\rfswarm\rfswam_manager>rfswarm -s Y:\robot-and-scenario\xpto2.rfs -n -a 2
Robot Framework Swarm: Manager
        Version 0.6.4
        Configuration File:  c:\users\dave\appdata\local\programs\python\python37\lib\site-packages\rfswarm_manager\RFSwarmManager.ini
Robot Framework Swarm: Manager
        Version 0.6.4
        Configuration File:  c:\users\dave\appdata\local\programs\python\python37\lib\site-packages\rfswarm_manager\RFSwarmManager.ini
<rfswarm.RFSwarmCore object at 0x0000025177492108>
```

**Comment by damies13** [comment]
The initial expected fix didn't work, but I've figured out a reasonable fix:

I created a dummy class to act as the entry point, this dummy class only contains an init function that sits in a loop with a 5 min sleep until the core on_closing is called, as on_closing calls sys.exit(0), the last sleep doesn't prevent exiting and the application exits cleanly.

This fix needed to be applied to both the manager and agent.

This branch has been merged into branch 0.6.5 and will be released with that version.
